website/content/docs/discovery/dns.mdx

```diff
@@ -136,17 +136,22 @@ it is recommended to use the HTTP API to retrieve the list of nodes.
 The format of a standard service lookup is:
 
 ```text
-[<tag>.]<service>.service[.<datacenter>].<domain>
+[<tag>.]<service>.service[.<datacenter-or-peer>].<domain>
 ```
 
-The `tag` is optional, and, as with node lookups, the `datacenter` is as
-well. If no tag is provided, no filtering is done on tag. If no
-datacenter is provided, the datacenter of this Consul agent is assumed.
+The `tag` is optional. If no tag is provided, no filtering is done on tag.
+
+The `datacenter-or-peer` is also optional.
```

Comment:
Intended to support in the future, but no support now since it's always parsed as dc